src/main/java/ua/vedroid/dao3/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,118 @@
+package ua.vedroid.dao3.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import ua.vedroid.dao3.dao.ManufacturerDao;
+import ua.vedroid.dao3.exceptions.DataProcessingException;
+import ua.vedroid.dao3.lib.Dao;
+import ua.vedroid.dao3.model.Manufacturer;
+import ua.vedroid.dao3.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers"
+                + "(manufacturer_name, manufacturer_country) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection
+                        .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+            }
+            resultSet.close();
```

Comment:
t is optional to close ResultSet, because when a Statement object is closed, its current ResultSet object, if one exists, is also closed.
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/Statement.html#close()